deepchem/models/dft/nnxc.py

```diff
@@ -0,0 +1,152 @@
+from abc import abstractproperty, abstractmethod
+from typing import Union
+try:
+    from dqc.xc.base_xc import BaseXC
+    from dqc.utils.datastruct import ValGrad, SpinParam
+    from dqc.api.getxc import get_xc
```

Comment:
This looks like a utility function. Can you copy this in directly from dqc?

Reply:
Are we trying to do that in this iteration? If so, I'm not so sure, but I will look into it.